plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestThriftHttpMetastoreClient.java

```diff
@@ -83,7 +83,27 @@ public void testHttpThriftConnection()
             ThriftMetastoreClientFactory factory = new HttpThriftMetastoreClientFactory(config, new TestingNodeManager(), OpenTelemetry.noop());
             URI metastoreUri = URI.create("http://localhost:" + metastoreServer.getPort());
             ThriftMetastoreClient client = factory.create(
-                    metastoreUri, Optional.empty());
+                    metastoreUri, Optional.empty(), Optional.empty());
+            assertThat(client.getAllDatabases()).containsExactly("testDbName");
+            assertThat(client.getDatabase("testDbName")).isEqualTo(new Database("testDbName", "testOwner", "testLocation", Map.of("key", "value")));
+            // negative case
+            assertThatThrownBy(() -> client.getDatabase("does-not-exist")).isInstanceOf(NoSuchObjectException.class);
+        }
+    }
+
+    @Test
+    public void testHttpThriftConnectionWithCustomCatalog()
+            throws Exception
+    {
+        ThriftHttpMetastoreConfig config = new ThriftHttpMetastoreConfig();
```

Comment:
Test not necessary in the context in which the http thrift connection can't get a custom catalog name.